core/commonMain/src/kotlinx/serialization/SerializationExceptions.kt

```diff
@@ -22,12 +22,14 @@ import kotlinx.serialization.descriptors.*
  * It is also an established pattern to validate input in user's classes in the following manner:
  * ```
  * @Serializable
- * class Foo(...) {
+ * class User(val age: Int, val name: String) {
  *     init {
- *         required(age > 0) { ... }
+ *         require(age > 0) { ... }
  *         require(name.isNotBlank()) { ... }
  *     }
  * }
+ *
+ * Json.decodeFromString<User>("""{"age": -100, "name": ""}""") // throws IAE from require()
```

Comment:
I think it's better to write the full name of the exception, because there is no such abbreviation anywhere else.